src/DB_Store.php

```diff
@@ -11,6 +11,26 @@
 
 class DB_Store extends ActionScheduler_Store {
 
+	/**
+	 * Valid database fields for this data store.
+	 *
+	 * @var array
+	 */
+	protected $db_fields = [
+		'action_id'            => 1,
+		'hook'                 => 1,
+		'status'               => 1,
+		'scheduled_date_gmt'   => 1,
+		'scheduled_date_local' => 1,
+		'args'                 => 1,
+		'schedule'             => 1,
+		'group_id'             => 1,
+		'attempts'             => 1,
+		'last_attempt_gmt'     => 1,
+		'last_attempt_local'   => 1,
+		'claim_id'             => 1,
+	];
```

Comment:
If you decide to add `ActionScheduler_Store::update_action()` as an abstract and have it be required in other data stores (as discussed in my other comment), then this should also be moved to `ActionScheduler_Store`. 

It will likely require a more generic name than `$db_fields` if that happens.